test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -1633,10 +1647,10 @@ public void testGetModifiedDocIds() throws Exception {
     executeUpdate("create table data(id integer, "
         + "other varchar(20) default 'hello, world', ts timestamp)");
     executeUpdate("insert into data(id, ts) values "
-        + "(1, dateadd('minute', 1, current_timestamp())),"
-        + "(2, dateadd('minute', 2, current_timestamp())),"
-        + "(3, dateadd('minute', 1, current_timestamp())),"
-        + "(4, dateadd('minute', -1, current_timestamp()))");
+        + "(1, " + dateAdd("minute", 1, "now()") + "),"
```

Comment:
This quoting is hard to read, and the quasi-SQL syntax mixed into Java syntax doesn't seem like a big win. Why not just a nowPlus(int minutes) method?